articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+Once you've provided both the client ID and client secret, you'll see an auto-generated **redirect URI**. Take note of this URL, since you'll need to provide this URI in your Auth0 Client Settings page in the Allowed Callback URLs section.
+
+::: note
+If you're using the resource owner password flow, you'll need to provide the **resource owner username** and **resource owner password** instead of the client ID and secret.
+:::
+
+When complete, click **Save** to persist your changes.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/new-server-saved.png)
+
+#### Set the Allowed Callback URL
+
+You'll need to provide the **redirect URI** that was auto-generated during the OAuth 2.0 authorization server setup process to Auth0. Log into the Management Dashboard, and navigate to **Clients**. Select your Client, and click **Settings**. Paste the URL into the **Allowed Callback URLs** field.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/set-callback-url.png)
+
+Click **Save**.
+
+### Authorize Your OAuth 2.0 Server for Use with Your API
+
+Before you can use Auth0 to secure your API, you'll need to set your API to use Auth0. You can do so using the Azure Publisher Portal.
+
+Begin by navigating to the APIs tab, and select the Basic Calculator API.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/api-list.png)
+
+Click over to the **Security** tab.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/security.png)
+
+Under **User Authorization**, select **OAuth 2.0**. In the new **Authorization Server** field that appears, select the server you configured in the previous step.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/set-auth0-as-authserver.png)
+
+Click **Save**.
+
+## Test Your Integration
+
+Navigate to the Developer Portal.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/developer-portal.png)
+
+Go to APIs > Basic Calculator (or the API you've created for this tutorial). 
+
+![](/media/articles/integrations/azure-api-mgmt/azure/dev-portal-apis.png)
+
+This opens up to the page where you can make a `GET` call that allows you to add two integers.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/dev-portal-calculator.png)
+
+Click Try It. This will bring up the page where you can provide the parameters for your call.
+
+Scroll down to the **Authorization** section. Next to the **Auth0** field, select **Authorization Code**.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/dev-portal-try-it.png)
+
+At this point, you'll see the Auth0 login widget in a popup window (if you don't, disable your popup blocker). Provide the credentials for the Auth0 user you created earlier in the tutorial, and sign in.
+
+![](/media/articles/integrations/azure-api-mgmt/azure/dev-portal-auth.png)
+
+If you were able to successfully sign in, you'll see a message appear with the expiration date of the access token you need to call the API.
```

Comment:
I had to search for some seconds at the screenshot, maybe highlight the message?